Resources/Locale/en-US/arcade/paradise.ftl

```diff
@@ -0,0 +1,4 @@
+﻿paradise-arcade-on-examine = [color=lightblue]{CAPITALIZE(SUBJECT($target))} {CONJUGATE-BE($target)} dreaming about paradise...[/color]
+paradise-arcade-window-title = Paradise Station
+paradise-arcade-button-label = Are you sure? This will connect you to another server.
```

Comment:
I believe this message may need to be more verbose so that this will close your current client.

Reply:
Done.